app/controllers/students/learning_reasources_controller.rb

```diff
@@ -0,0 +1,16 @@
+module Students
+  class LearningReasourcesController < ApplicationController
```

Comment:
isn't this supposed to be `LearningResource` check for typos

Reply:
You're right! my bad, thanks.